node/core/candidate-validation/src/lib.rs

```diff
@@ -638,11 +638,11 @@ trait ValidationBackend {
 		executor_params: ExecutorParams,
 	) -> Result<WasmValidationResult, ValidationError> {
 		// Construct the PVF a single time, since it is an expensive operation. Cloning it is cheap.
```

Comment:
Is this comment still true? The `Pvf` is behind an `Arc`, but not the executor params. Should they be, also?

Reply:
Sorry for giving you so much work with all my comments. 😅 Probably we should focus on getting the PR through, and leave the minor comments for a follow-up.

Reply:
Hmmm, I'm probably not in the best condition rn to design a better solution... The whole thing ends up in `ToHost::ExecutePvf` that gets SCALE-encoded and sent through overseer anyway, so it sounds like it's not needed to be passed by value at all, but it also gets encapsulated a couple of times by its way...
No, I'm done. I'll think about that next year :zany_face:

Reply:
Okay, so for now, I've just put it under the `Arc` reference, which mitigates cloning overhead; that was an easy change. Let's think more about that in the scope of future refactoring. I want to believe we could make it so we wouldn't ever need to move `ExecutorParams` out, just borrowing it where needed, but that requires careful design.

Reply:
Nice, thanks! Feel free to resolve.